Server/src/compile.sh

```diff
@@ -1,14 +1,18 @@
 # Author: Pontus Laestadius
-# Version: 1.0
+# Version: 1.2
```

Comment:
Why did the version change from 1.0 to 1.2

Reply:
+1

Reply:
+1